WooCommerce/Classes/ViewRelated/Orders/FulfillViewController.swift

```diff
@@ -510,7 +520,13 @@ private extension FulfillViewController {
             return Section(title: title, secondaryTitle: nil, rows: rows)
         }()
 
-        let addTracking: Section = {
+        let addTracking: Section? = {
+            // Hide the section if we consider the shipment
+            // tracking plugin is not installed
```

Comment:
Would it be okay to re-word this? "Hide this section if the shipment tracking plugin is not installed"